### ISSUE TITLE
Include Camel stack

### ISSUE THREAD

**Issue by apupier** [redhat-developer/rh-che#921]
**Issue problem:** 
Camel stack has been added recently to Eclipse Che. it would be nice to have it available in rh-che as it is an upstream project of Red Hat Fuse.
see https://github.com/eclipse/che/blob/6f30e14113dc34c99240e08d1558b9dddc563176/ide/che-core-ide-stacks/src/main/resources/stacks.json#L1959

**Red Hat Che version:** 

version: (help/about menu)

 - [ ] I can reproduce it on latest official image

**Reproduction Steps:**

Describe how to reproduce the problem

**Runtime:**

runtime used:
   - [ ] minishift (include output of `minishift version`)
   - [ ] OpenShift.io
   - [ ] Openshift Container Platform (include output of `oc version`)
   - [ ] k8s (include output of `kubectl version`)

**Comment by ibuziuk** [comment]
@apupier maybe you would be able to provide a PR to rh-che - https://github.com/redhat-developer/rh-che/blob/master/assembly/fabric8-stacks/src/main/resources/stacks.json ?

**Comment by apupier** [comment]
@ibuziuk the stacks.json are duplicated between che and rh-che? Every time there will be an update in Eclipse Che, we will need to report it here? tests written in Eclipse Che codebase needs to be duplicated too?

**Comment by ibuziuk** [comment]
@apupier yes, in rh-che we have a custom version of stacks, so changes will need to be backported. In terms of test for stacks,  those should be part of upstream project since we are building rh-che against the fixed upstream version and just add extra osio specific things.

> Same id needs to be used for the stack or a different one?
yes

**Comment by apupier** [comment]
> we are building rh-che against the fixed upstream version

Camel stack has been provided for Che 6.12.
The current fixed version used here is 6.11.1, should we wait for 6.12.0 used in rh-che?
In theory, there is no reason that the Camel stack is not working with 3.11.1 but tests were not launched with it.

**Comment by apupier** [comment]
I'm noticing that "com.redhat.oc-login" has been added to several stacks here (but not all) compared to upstream Che.
What is it used for? How to know if i need to add it also for Camel stack?

**Comment by ibuziuk** [comment]
@apupier this one is used for oc login to the user namespace on OSO and possibility to execute `oc` commands from terminal. So, I believe you could update image to `registry.devshift.net/che/spring-boot` [1] which includes oc binaries and add the oc login installer to the stack.

[1] https://github.com/redhat-developer/che-dockerfiles/blob/master/recipes/dockerfiles/spring-boot/Dockerfile
